Std/Tactic/CoeExt.lean

```diff
@@ -5,10 +5,43 @@ Authors: Gabriel Ebner, Mario Carneiro
 -/
 import Lean.PrettyPrinter.Delaborator.Builtins
 import Std.Lean.Delaborator
-open Lean Meta Std
+open Lean Elab.Term Meta Std
+
+/-!
+# The `@[coe]` attribute, used to delaborate coercion functions as `↑`
+
+When writing a coercion, if the pattern
+```
+@[coe]
+def A.toB (a : A) : B := sorry
+
+instance : Coe A B where coe := A.toB
+```
+is used, then `A.toB a` will be pretty-printed as `↑a`.
+
+This file also provides `⇑f` and `↥t` notation, which are syntax for `Lean.meta.coerceToFunction?`
+and `Lean.Meta.coerceToSort?` respectively.
+-/
 
 namespace Std.Tactic.Coe
 
+/-- `⇑ t` coerces `t` to a function. -/
+-- the precendence matches that of `coeNotation`
+elab:1024 (name := coeFunNotation) "⇑" m:term:1024 : term => do
```

Comment:
Is there a reason to use 1024 explicitly instead of `max`?

Reply:
I just copied core, which also writes 1024 explicitly